godtools/Controllers/LanguagesViewController.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  LanguagesViewController.swift
+//  godtools
+//
+//  Created by Ryan Carlson on 4/18/17.
+//  Copyright © 2017 Cru. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class LanguagesViewController: UIViewController {
+    
+    let languagesManager = LanguagesManager.shared
+    
+    @IBOutlet weak var tableView: UITableView! {
+        didSet {
+            tableView.delegate = languagesManager
+            tableView.dataSource = languagesManager
+        }
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        languagesManager.loadFromDisk().catch(execute: { error in
```

Comment:
This is really cool! Would you please move each load to a separate method? Also, I believe we should create something for error handle, in order to make it more easy and automatic